### PR TITLE
Adds script to run rails c on Elastic Beanstalk

### PR DESCRIPTION
**Pull request by mdesanti**
## Summary

Adds a script that makes it easy to run `rails c` in Elastic Beanstalk

### CODE REVIEW

File: docs/deploy.rb [hunk 1]
```diff
@@ -38,6 +38,8 @@
 
 You can find all the logs under the `/var/logs` folder.
 
+To run `rails c` first connecto to the EC2 instance and `cd /var/app/current`. Then run `script/aws-console`
```
Comment (Dinuuu) [inline docs/deploy.rb]: typo in `connect`

**Comment by mdesanti** [comment]
@Dinuuu fixed

File: docs/deploy.rb [hunk 2]
```diff
@@ -38,6 +38,8 @@
 
 You can find all the logs under the `/var/logs` folder.
 
+To run `rails c` first connec to to the EC2 instance and `cd /var/app/current`. Then run `script/aws-console`
```
Comment (francoprud) [inline docs/deploy.rb]: Typo again haha

Comment (Dinuuu) [inline docs/deploy.rb]: Hahahah

**Comment by mdesanti** [comment]
@Dinuuu @francoprud

**Pull request merged**